scss/_select2.scss

```diff
@@ -1,35 +1,91 @@
-.select2-container--bootstrap {
+$component-name: select2 !default;
 
-  .select2-dropdown {
-    padding: $input-padding-x $input-padding-y;
-    border-radius: 0;
-  }
+.#{$component-name}-container--bootstrap {
+  .#{$component-name} {
+    &-dropdown {
+      padding: $input-padding-x $input-padding-y;
+      border-top: 1px solid;
```

Comment:
I still don't understand the use for `rem` over `px` but we should be consistent (Bootstrap uses `rem` everywhere).

Reply:
They don't use `rem` everywhere: https://github.com/twbs/bootstrap/blob/master/scss/_variables.scss#L334
https://github.com/twbs/bootstrap/commit/ac5be12f24ec864a1a1d633942b6e09415fc2f18

I like this one: https://medium.com/@kilgarenone/in-short-when-to-use-px-em-and-rem-b2de94e1b829

Reply:
Ok let's use px for borders.